src/common/Element.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Base class for all elements
+ */
+import Props from "../common/Props.js";
+import defineFormAssociated from "./defineFormAssociated.js";
+import defineEvents from "./defineEvents.js";
+
+const Self = class NudeElement extends HTMLElement {
+	static postConstruct = [];
+	#initialized = false;
+
+	constructor () {
+		super();
+		this.constructor.init();
+
+		if (this.propChangedCallback && this.constructor.props) {
+			this.addEventListener("propchange", this.propChangedCallback);
+		}
+
+	}
+
+	connectedCallback () {
+		if (!this.#initialized) {
+			this.initializeProps?.();
+
+			this.#initialized = true;
+		}
+	}
+
+
+	static init () {
+		// One time stuff
+		if (this._initialized) {
```

Comment:
Is it some sort of static variable that shouldn't be defined? I don't understand how we can use uninitialized variables in static methods. I didn't find it `_initialized` in other parts of the code base. Does the `HTMLElement` class (instance) have it?

Reply:
Got it. Thanks!